src/links/v1/providers.json

```diff
@@ -63,6 +63,17 @@
         "link_type": "external"
       }
     },
+    {
+      "type": "links",
+      "id": "MatCloud",
+      "attributes": {
+        "name": "An integrated high-throughput computational materials platform",
+        "description": "MatCloud is a high-throughput computing platform integrating data, simulation and supercomputing.",
+        "base_url": "http://207.246.86.9:5000/optimade",
```

Comment:
You don't have a named domain to put in here instead? As otherwise you will have to change this again later once you get that.

Furthermore, The OpenAPI schema shown there does not reflect what would be expected from the named endpoints. See the specification [here](https://github.com/Materials-Consortia/OPTIMADE/blob/develop/optimade.rst) to understand better how to setup your OPTIMADE implementation (e.g., endpoint names should consist of lowercase alphanumerics and usually be in the plural form, e.g., `/structures` instead of `/Structure`).

Reply:
Our website is still under construction, and only some of the functions of the optimade specification have been implemented. And for these irregularities, I will carefully check and modify. 

Later, the domain name will be unified under MatCloud (http://matcloud.cnic.cn/).
Thank you for your suggestion.